### PR TITLE
Simplify bootstrap

### PR DESCRIPTION
**Pull request by YaronMiro**
@amitaibu 
The demo is finished but the post text is still at a draft stage.
I'm still struggling with it :) I hope that tomorrow morning I will have a better inspiration...

### CODE REVIEW

File: app/_posts/2015-11-01-simplify-bootstrap.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+permalink: "/content/simplify-bootstrap"
+layout: post
+author: YaronMiro
+published: true
+---
+
+{% include setup %}
+
+# Simplifying bootstrap responsive design
+I want to share with my experience and insights about responsive web design using
```
Comment (bricel) [inline app/_posts/2015-11-01-simplify-bootstrap.md]: with => with you

File: app/_posts/2015-11-01-simplify-bootstrap.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Twitter bootstrap framework and hopefully help you to simplify your workflow and
```
Comment (bricel) [inline app/_posts/2015-11-01-simplify-bootstrap.md]: bootstrap => Bootstrap

File: app/_posts/2015-11-01-simplify-bootstrap.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<!-- more -->
+
+{% include demo_block.html demo='http://yaronmiro.github.io/bootstrap-responsive/' code='https://github.com/YaronMiro/bootstrap-responsive' %}
+
+## The traditional approach
+Bootstrap provides us with a powerful responsive layout and a matching grid system
+but with **great power comes great responsibility!**. The responsibility of making
+sure that each breakpoint layout looks consist and not broken isn't always an easy task
+to achieve and above all it consume a considerable amount of project time.
+The vast majority of our project at gizra are based upon bootstrap
```
Comment (bricel) [inline app/_posts/2015-11-01-simplify-bootstrap.md]: gizra => Gizra 
bootstrap => B..

**Comment by YaronMiro** [comment]
* Explain when to choose the default breakpoints and when we need less breakpoints.
* The main idea is that responsive design is a major "time & effort consumer" - the best practice is to adjust the amount of your website responsive with your website needs and also your visitors needs - if a visitor came from a mobile device or a desktop device (e.g: checking with Google Analytics)

**Comment by YaronMiro** [comment]
@amitaibu Ready for review.

**Comment by YaronMiro** [comment]
@amitaibu 
This the link: http://gfycat.com/DistortedBarrenGroundbeetle to the gif image. I already replace the older (7mb) with this new one(3m). 

* I liked the changes you have added :)
* I have updated the demo - according to what we have discussed.

**Comment by amitaibu** [comment]
@niryariv please review

**Comment by amitaibu** [comment]
I have temporarily disabled the VR tests.

Merged, and deployed - thanks!

http://www.gizra.com/content/mindful-custom-breakpoints-bootstrap/

**Pull request closed without merging**